boringtun/src/noise/timers.rs

```diff
@@ -289,7 +289,7 @@ impl Tunn {
                     && now - aut_packet_received >= KEEPALIVE_TIMEOUT + REKEY_TIMEOUT
                     && timers.want_handshake.swap(false, Ordering::Relaxed)
                 {
-                    tracing::debug!("HANDSHAKE(KEEPALIVE + REKEY_TIMEOUT)");
+                    tracing::error!("HANDSHAKE(KEEPALIVE + REKEY_TIMEOUT)");
```

Comment:
I think this should be a warning because a single packet drop could cause this, so it would not be unexpected, and is not indicative of an overall significant failure in the system.